AI-and-Analytics/Features-and-Functionality/IntelTensorFlow_InferenceOptimization/sample.json

```diff
@@ -0,0 +1,20 @@
+{
+	"guid": "9d32f194-8667-41d3-865d-d43e9983c471",
+	"name": "Tutorial : Optimize TensorFlow pre-trained model for inference",
```

Comment:
Why are we adding a tutorial to the name, usually this is for organizational purposes, if that's the case are you planning on modifying the other samples?

Reply:
changed it accordingly.